app/templates/components/nypr-brick-item.hbs

```diff
@@ -1,4 +1,4 @@
-<a href="{{item.attributes.url}}" class="brick_row__item {{item.attributes.template}} item--trending"
+<div class="brick_row__item {{item.attributes.template}} item--trending"
```

Comment:
What's the purpose of the single `_`? I thought it was a double `__` to indicate a nested element and `-` for naming elements or blocks, right?